site/content/docs/02-template-syntax.md

```diff
@@ -0,0 +1,592 @@
+
+<!-- grouped checkbox inputs populate an array -->
+<input type="checkbox" bind:group={fillings} value="Rice">
+<input type="checkbox" bind:group={fillings} value="Beans">
+<input type="checkbox" bind:group={fillings} value="Cheese">
+<input type="checkbox" bind:group={fillings} value="Guac (extra)">
+```
+
+---
+
+A `<select>` value binding corresponds to the `value` property on the selected `<option>`, which can be any value (not just strings, as is normally the case in the DOM).
+
+```html
+<select bind:value={selected}>
+	<option value={a}>a</option>
+	<option value={b}>b</option>
+	<option value={c}>c</option>
+</select>
+```
+
+---
+
+A `<select multiple>` element behaves similarly to a checkbox group.
+
+```html
+<select multiple bind:value={fillings}>
+	<option value="Rice">Rice</option>
+	<option value="Beans">Beans</option>
+	<option value="Cheese">Cheese</option>
+	<option value="Guac (extra)">Guac (extra)</option>
+</select>
+```
+
+---
+
+When the value of an `<option>` matches its text content, the attribute can be omitted.
+
+```html
+<select multiple bind:value={fillings}>
+	<option>Rice</option>
+	<option>Beans</option>
+	<option>Cheese</option>
+	<option>Guac (extra)</option>
+</select>
+```
+
+---
+
+Media elements (`<audio>` and `<video>`) have their own set of bindings — four *readonly* ones...
+
+* `duration` (readonly) — the total duration of the video, in seconds
+* `buffered` (readonly) — an array of `{start, end}` objects
+* `seekable` (readonly) — ditto
+* `played` (readonly) — ditto
+
+...and three *two-way* bindings:
+
+* `currentTime` — the current point in the video, in seconds
+* `paused` — this one should be self-explanatory
+* `volume` — a value between 0 and 1
+
+```html
+<video
+	src={clip}
+	bind:duration
+	bind:buffered
+	bind:seekable
+	bind:played
+	bind:currentTime
+	bind:paused
+	bind:volume
+></video>
+```
+
+---
+
+Block-level elements have readonly `clientWidth`, `clientHeight`, `offsetWidth` and `offsetHeight` bindings, measured using a technique similar to [this one](http://www.backalleycoder.com/2013/03/18/cross-browser-event-based-element-resize-detection/).
+
+```html
+<div bind:offsetWidth={width} bind:offsetHeight={height}>
+	<Chart {width} {height}/>
+</div>
+```
+
+---
+
+To get a reference to a DOM node, use `bind:this`.
+
+```html
+<script>
+	import { onMount } from 'svelte';
+
+	let canvasElement;
+
+	onMount(() => {
+		const ctx = canvasElement.getContext('2d');
+		drawStuff(ctx);
+	});
+</script>
+
+<canvas bind:this={canvasElement}></canvas>
+```
+
+
+### Component bindings
+
+* `bind:property={value}`
+
+You can bind to component props using the same mechanism.
+
+```html
+<Keypad bind:value={pin}/>
+```
+
+---
+
+Components also support `bind:this`, allowing you to interact with component instances programmatically.
+
+(Note that we can do `{cart.empty}` rather than `{() => cart.empty()}`, since there is no `this` inside a component's `<script>` block.)
```

Comment:
I had to read this twice to realize what it's talking about. "Since there is no this" sounded like a bit of a non-sequitur at first. I would be more explicit and say something about not having to worry about binding or closures. This also seems like an odd location for this comment anyway.

Reply:
I was in two minds as well, but I figured it made sense since it's likely to be the only place we call a component method in this way, and experienced JS devs would likely do a double take upon seeing an 'instance method' dealt with like that. Maybe it should be a blockquote instead, so it feels like more of an aside?

Tweaked the wording a bit

Reply:
I do like the new wording better. And I also think that a blockquote aside would make this smoother.

Reply:
done